### ISSUE TITLE
Support command line option "-listen"

### ISSUE THREAD

**Issue by alkisg** [bk138/multivnc#135]
Hi, I installed multivnc in Ubuntu 18.04 and I tried:

```shell
multivnc -listen
```

I got a dialog stating "Failure connecting to server at -listen!". Is there a command line argument available to launch multivnc in listen mode? If not, is it possible to add one, preferably one that only shows the UI when the first connection is received?

Then I went on with launching plain `multivnc` and clicking on the "Listen" toolbar button. I was able to connect a client there with `x11vnc -connect multivnc-ip`. But I wasn't able to connect a second client simultaneously, like in `xvnc4viewer -listen`. Is this possible? If not, could it be supported in a future version?

If these features are met, multivnc will be an excellent first choice as a dependency of epoptes.org (I'm the main developer there).
Thank you!

**Comment by bk138** [comment]
No, there's none and yes, that possible to add. For the listen-for-multiple-servers, would you be so kind and open a separate issue?

**Comment by alkisg** [comment]
Thank you; I filed #136 for the multiple reverse connections.

Regarding the `platform-linux` tag, it would be great if `-listen` was also supported in Windows, as there we could replace realvnc with multivnc.

**Comment by bk138** [comment]
As it stands now, I'm not able to invest a substantial amount of time in this due to childcare, but I can set apart some time to do pull request reviews.

**Comment by alkisg** [comment]
> I'm not able to invest a substantial amount of time

Unfortunately that's the case for me as well, and it's the main reason that epoptes relies on external programs instead of directly using libvnc. :/

Thank you for your time!
